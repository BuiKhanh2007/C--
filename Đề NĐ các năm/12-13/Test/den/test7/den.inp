63 247
32 94
139 276
124 187
113 323
123 307
164 392
33 238
176 269
3 134
115 183
48 98
160 328
35 288
155 200
98 323
64 310
57 80
22 54
