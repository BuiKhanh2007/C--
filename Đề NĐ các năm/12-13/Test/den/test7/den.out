2249
628
1849
756
2842
2532
3293
2415
1333
1301
795
524
2387
3139
575
3021
3206
232
316
